ProcessMaker/Http/Controllers/Api/V1_1/TaskController.php

```diff
@@ -56,6 +57,10 @@ public function index()
 
     private function processFilters(Request $request, Builder $query)
     {
+        if (request()->has('user_id')) {
```

Comment:
Consider to use `$request` instead of `request()` to keep consistency with line 64.

Reply:
It was fixed